spec/integration/outputs/parent_spec.rb

```diff
@@ -104,6 +104,12 @@
             }
           }
         }
+        if ESHelper.es_version_satisfies?('<6')
+          mapping.merge!({
+                 "settings" => {
+                   "mapping.single_type" => true
```

Comment:
This setting is only valid in ES 5.6 and above, (it will fail on 5.5 and below, right?)

Reply:
@jordansissel This clause is within an outer satisfies?('>=5.6'), so it won't run on 5.5 and below

Reply:
Ahh yes. Github's diff wasn't showing that until I expanded. Thanks!